pinot-common/src/main/java/org/apache/pinot/common/data/objects/TextObject.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.pinot.common.data.objects;
+
+import java.util.List;
+
+import org.apache.pinot.common.data.PinotObject;
+
+import com.google.common.collect.Lists;
+
+public class TextObject implements PinotObject {
```

Comment:
What's the difference between Text and String?

Reply:
Text is a long string. It can give us some hints such as don't do dictionary encoding. More like TEXT vs VARCHAR in MySQL